tests/Doctrine/Tests/Common/Cache/ChainCacheTest.php

```diff
@@ -68,6 +68,51 @@ public function testFetchPropagateToFastestCache() : void
         self::assertTrue($cache1->contains('bar'));
     }
 
+    public function testFetchPropagatesToFastestCacheUsingDefaultLifeTimeForDownstreamCacheProviders() : void
+    {
+        $defaultLifeTimeForDownstreamCacheProviders = 0;
+
+        $cache1 = $this
+            ->getMockBuilder(ArrayCache::class)
+            ->setMethods(['doSave'])
+            ->getMock();
+        $cache1
+            ->expects($this->once())
+            ->method('doSave')
+            ->with('[bar][1]', 'value', $defaultLifeTimeForDownstreamCacheProviders);
+        $cache2 = new ArrayCache();
+        $cache2->save('bar', 'value');
+
+        $chainCache = new ChainCache([$cache1, $cache2]);
+
+        $result = $chainCache->fetch('bar');
+
+        self::assertEquals('value', $result);
+    }
+
+    public function testFetchPropagatesToFastestCacheUsingIndicatedDefaultLifeTimeForDownstreamCacheProviders() : void
+    {
+        $pecificDefaultLifeTimeForDownstreamCacheProviders = 12345;
```

Comment:
Did you really mean to use that `$` sign as an S? :sweat_smile:

Reply:
Haha, oops, nope, will change!